server/lib/insightProviders/pageInsightProviders/googlePageSpeedInsights.js

```diff
@@ -19,11 +20,11 @@ module.exports = function googlePageSpeedInsights(url) {
 			link: `https://developers.google.com/speed/pagespeed/insights/?url=${url}&tab=mobile`
 		},{
 			name: 'WeightOfResources',
-			value: parseInt(results.pageStats.htmlResponseBytes, 10) + 
-					parseInt(results.pageStats.cssResponseBytes, 10) + 
-					parseInt(results.pageStats.imageResponseBytes, 10) + 
-					parseInt(results.pageStats.javascriptResponseBytes, 10) + 
-					parseInt(results.pageStats.otherResponseBytes, 10),
+			value: parseInt(results.pageStats.htmlResponseBytes, 10) || 0 + 
```

Comment:
Occasionally, the values returned from Google couldn't be parsed as integers (I can't remember what was being returned off the top of my head), so they would become NaN. If a value returned is parsed as NaN, we assign that value to 0, so we can still store the rest of the metrics and not cause our SQL server to error when we try to input an NaN into a field expecting an int.